components/developer/clang-12/Makefile

```diff
@@ -0,0 +1,118 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Aurelien Larcher.  All rights reserved.
+# Copyright 2019 Alexander Pyhalov
+# Copyright 2019 Michal Nowak
+# Copyright 2019 Till Wegmüller
+#
+
+BUILD_BITS=		64
+BUILD_STYLE=		cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		clang-12
+LLVM_NAME=		llvm
+CLANG_NAME=		clang
+
+COMPONENT_VERSION=	12.0.0
+COMPONENT_REVISION=	1
+COMPONENT_MAJOR_VERSION= \
```

Comment:
You don't need to set COMPONENT_REVISION if we never have released the version before. It's an artificial discriminator to be able to change our version if the version of project hasn't changed.

Reply:
oh, alright